base_binary_url_import/tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+# from . import tes_binary_url_import
```

Comment:
Why is the test deactivated?

Reply:
I will remove it. I don't have this test in the module